movement/watch_faces/complication/sunrise_sunset_face.h

```diff
@@ -53,6 +53,7 @@ typedef struct {
     uint8_t active_digit;
     bool location_changed;
     watch_date_time rise_set_expires;
+    uint8_t longLatToUse;
```

Comment:
please add new members to the end of the struct

Reply:
done